src/Delegates/HandlesSystem.php

```diff
@@ -24,6 +24,17 @@ public function health(): ?array
         return $this->health->show();
     }
 
+    public function isHealthy(): bool
+    {
+        try {
+            $health = $this->health->show();
```

Comment:
```suggestion
            $this->health->show();
```
Not sure you need to use a variable.